.devcontainer/devcontainer.json

```
@@ -0,0 +1,14 @@
+{
+  "dockerFile": "Dockerfile",
+  "extensions": [
+    "akamud.vscode-theme-onelight",
+    "bierner.github-markdown-preview",
+    "denoland.vscode-deno",
+    "eamodio.gitlens",
+    "ms-vscode.theme-1337",
+    "streetsidesoftware.code-spell-checker",
+    "tyriar.sort-lines",
+    "usernamehw.errorlens"
+  ],
```

Let's keep the devcontainer as clean as possible with only the necessary extensions
```suggestion
  "extensions": [
    "denoland.vscode-deno",
  ],
```

---

I've trimmed out a few items, but I prefer to keep a more full-featured .devcontainer because in the past, as a minor contributor to other projects, I've appreciated being able to jump in, make some changes in a reasonably-capable environment, and jump out, without spending time yak shaving.

I'm open to changing it later.